### DISCUSSION TITLE
ETA for training code release

### DISCUSSION THREAD

**Discussion by TheUnamusedFox** [tiiuae/Falcon-Perception#18] [General]
Hello! I was just wondering if there is any kind of eta on release of training code? I have a small dataset of masks for various characters in moderately to heavily stylized images that I wanted to try training a lora for. Tried to throw something together with assistance of claude, codex, and the paper, but I don't personally know enough about training outside of image diffusion to fix llm errors and assumptions.
